### ISSUE TITLE
[Bug] Data field has changed when create a merge request

### ISSUE THREAD

**Issue by lihaorong840506** [yuntijs/yunti#15]
### 💻 系统环境 | Operating System

Windows

### 🐛 问题描述 | Bug Description

API:
# Write your query or mutation here
```gql
mutation createMergeRequest($mergeRequestInput: MergeRequestInput!) {
  createMergeRequest(mergeRequestParam: $mergeRequestInput) {
    sourceBranchName,
    targetBranchName,
    title,
    description,
    options,
    mergeRequestStatus,
    createAt,
    updateAt,
  }
}
```
Before query variables:
```json
{ 
  "mergeRequestInput":
  {
    "source_branch":"app-lfon8/user-0ekq4/tttt2",
    "target_branch":"app-lfon8/user-0ekq4/ttt",
    "title":"tttt",
    "description":"ttttt",
    "assignee_id":"user-0ekq4",
    "options": "{\"delSourceBranch\":1}"
  }
}
```

Required query variables:
```json
{ 
  "mergeRequestInput":
  {
    "sourceBranch":"app-lfon8/user-0ekq4/tttt2",
    "targetBranch":"app-lfon8/user-0ekq4/ttt",
    "title":"tttt",
    "description":"ttttt",
    "assigneeId":"user-0ekq4",
    "options": "{\"delSourceBranch\":1}"
  }
}
```

### 🚦 期望结果 | Expected Behavior

```json5
{ 
  "mergeRequestInput":
  {
    "sourceBranch":"app-lfon8/user-0ekq4/tttt2",
    "targetBranch":"app-lfon8/user-0ekq4/ttt",
    "title":"tttt",
    "description":"ttttt",
    "assigneeId":"user-0ekq4",
    "options": "{\"delSourceBranch\":1}"  // new added
  }
}
```

### 📝 补充信息 | Additional Information

"options": "{\"delSourceBranch\":1}":

delSourceBranch: 1 => delete source branch after merged.

delSourceBranch: 0 => nothing to do after merged.

**Comment by lihaorong840506** [comment]
#14

**Comment by Carrotzpc** [comment]
@zggmd try @yuntijs/yunti-bff-sdk@1.0.1

**Comment by Carrotzpc** [comment]
resolved in #18
